apps/admin/backend/src/app.ts

```diff
@@ -891,6 +905,35 @@ function buildApi({
       });
     },
 
+    async printWriteInAdjudicationReport(): Promise<void> {
+      return printWriteInAdjudicationReport({
+        store,
+        printer,
+        logger,
+        userRole: assertDefined(await getUserRole()),
+      });
+    },
+
+    async exportWriteInAdjudicationReportPdf(input: {
+      path: string;
+    }): Promise<ExportDataResult> {
+      return exportWriteInAdjudicationReportPdf({
+        path: input.path,
+        store,
+        logger,
+        userRole: assertDefined(await getUserRole()),
+      });
+    },
+
+    async getWriteInAdjudicationReportPreview(): Promise<string> {
```

Comment:
Are we writing the PDF to disk because of a Grout limitation? Can we just use the browser's built-in PDF rendering rather than going to great lengths to incorporate pdfjs? Returning a `Buffer` wouldn't work because Grout turns it into something the browser can't just use. Perhaps its time we added an escape hatch to Grout to allow returning a `Response` that directly controls the HTTP response.

Reply:
FWIW I'm also working on PDF ballot previews and I've been experimenting with just sending the Buffer via grout. It seems to work pretty well.

I think using the browser's built-in PDF rendering would be cool, but from what I've seen you can't customize the UI at all. In Chrome there's a toolbar with a bunch of extra functionality (e.g. printing, downloading) that we probably don't want.

Reply:
I only went with the file approach because I'm already introducing loading latency in this PR, and I thought that the base64 encoding and decoding would make it even slower. But I was probably overthinking it, a `Buffer` should be fine and easier to manage. 

I'm hesitant about using built-in functionality because I think that would continue to tie us to a specific browser rather than a generic, simple renderer. My understanding is that different browsers roll their own PDF interface and they are different, and something like WebKitGTK might not have one at all.